src/arkit.cs

```diff
@@ -278,6 +287,55 @@ interface ARPlaneAnchor {
 			[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
 			get;
 		}
+
+		[iOS (11,3)]
+		[Export ("geometry", ArgumentSemantic.Strong)]
+		ARPlaneGeometry Geometry { get; }
+	}
+
+	[iOS (11,3)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface ARPlaneGeometry : NSSecureCoding {
+		[Export ("vertexCount")]
+		nuint VertexCount { get; }
+
+		[EditorBrowsable (EditorBrowsableState.Advanced)]
+		[Export ("vertices")]
+		IntPtr GetRawVertices ();
+
+		[Export ("textureCoordinateCount")]
+		nuint TextureCoordinateCount { get; }
+
+		[EditorBrowsable (EditorBrowsableState.Advanced)]
+		[Export ("textureCoordinates")]
+		IntPtr GetRawTextureCoordinates ();
+
+		[Export ("triangleCount")]
+		nuint TriangleCount { get; }
+
+		[EditorBrowsable (EditorBrowsableState.Advanced)]
+		[Export ("triangleIndices")]
+		IntPtr GetRawTriangleIndices ();
+
+		[Export ("boundaryVertexCount")]
+		nuint BoundaryVertexCount { get; }
+
+		[EditorBrowsable (EditorBrowsableState.Advanced)]
+		[Export ("boundaryVertices")]
+		IntPtr GetRawBoundaryVertices ();
+	}
+
+	[iOS (11,3)]
+	[BaseType (typeof(SCNGeometry))]
+	interface ARSCNPlaneGeometry {
+		[Static]
+		[Export ("planeGeometryWithDevice:")]
+		[return: NullAllowed]
+		ARSCNPlaneGeometry CreatePlaneGeometry (IMTLDevice device);
```

Comment:
Why not just `Create`? you already know it is a `PlaneGeometry` because of the type name.

Reply:
Mmh that was good feedback for https://github.com/xamarin/xamarin-macios/pull/2681 when `ARSCNFaceGeometry.CreateFaceGeometry` was introduced (:

I agree though, `PlaneGeometry` should be implied by the type's name.

I'll make `CreateFaceGeometry` `[Obsolete]` and move it to `Create` too for API consistency.